src/main/java/de/tum/cit/aet/artemis/core/service/ArtemisSuccessfulLoginService.java

```diff
@@ -71,7 +75,9 @@ public ArtemisSuccessfulLoginService(UserRepository userRepository, MailSendingS
      * Handles successful authentication events.
      * Sends a login notification email to users when they successfully authenticate.
      *
-     * @param username the username of the user who has successfully logged in
+     * @param username             the username of the user who has successfully logged in
+     * @param authenticationMethod the method used for authentication (e.g., password, passkey, SAML2)
+     * @param clientEnvironment    the environment information of the client (optional)
      */
```

Comment:
Can we rely on javadoc features instead of hardcoding here? 
```suggestion
     * @param username             the username of the user who has successfully logged in
     * @param authenticationMethod the method used for authentication
     * @param clientEnvironment    the environment information of the client (optional)
     * @see AuthenticationMethod for available authentication methods
     */
```